README.md

```diff
@@ -19,6 +19,7 @@ _Note: 🔶 denotes projects written in Swift._
 - [Color Picker Pro](https://github.com/oscardelben/Color-Picker-Pro) - Color Detection Tool for Designers and Developers.
 - [SketchToolbox](https://github.com/buzzfeed/Sketch-Toolbox) - A plugin manager for Sketch.app.
 - [Spectacle](https://github.com/eczarny/spectacle) - Spectacle allows you to organize your windows without using a mouse.
+- [Swift Random](https://github.com/thellimist/SwiftRandom) - A tiny generator of random data for swift
```

Comment:
Thank you for adding your project @thellimist, can you add your project to the bottom, on line 32? The "*items are listed in order of submission*"